### PR TITLE
Vale Accept List Improvements

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Review by makkes — APPROVED** [review]

**Comment by makkes** [comment]
Thanks!!

File: .github/styles/Vocab/Docs/accept.txt [hunk 1]
```diff
@@ -29,6 +30,7 @@ Kafka
 Kaptain
 Katib
 Keras
+keyspace
 Kibana
 Knative
 Kommander
```
Comment (gracedo) [inline .github/styles/Vocab/Docs/accept.txt:36]: Can we also add `Kubecost` in this list?

Comment (gracedo) [reply .github/styles/Vocab/Docs/accept.txt:36]: also `kubeconfig`

Comment (makkes) [reply .github/styles/Vocab/Docs/accept.txt:36]: I added them.

**Review by gracedo — APPROVED** [review]

**Review by samvantran — APPROVED** [review]

**Comment by cmastr** [comment]
Closing in favor of #3246

**Pull request closed without merging**